src/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -12,6 +12,7 @@
 from turicreate.toolkits.object_detector.object_detector import ObjectDetector as _ObjectDetector
 from turicreate.toolkits.one_shot_object_detector.util._augmentation import preview_synthetic_training_data as _preview_synthetic_training_data
 import turicreate.toolkits._internal_utils as _tkutl
+from turicreate.toolkits._internal_utils import _toolkit_serialize_summary_struct
```

Comment:
This is fine, don't worry about changing it. If it's only being used in one function, I would just import it at the start of that function rather than for the whole file.

Reply:
good!